<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bob</class>
 <widget class="QMainWindow" name="Bob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>476</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="up_line">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bob : Public key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="publickey_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::LineWrapMode::WidgetWidth</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exchange_button">
        <property name="text">
         <string>Exchange Public Keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decrypt_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Decrypt Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showmessage_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Message Decrypted</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="middle_line">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>You received : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="message_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="log_line">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Log messages</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
